core/templates/dev/head/profile/Preferences.js

```diff
@@ -35,6 +35,10 @@ oppia.controller('Preferences', ['$scope', '$http', '$rootScope', '$modal', '$ti
     _saveDataItem('user_bio', userBio);
   };
 
+  $scope.saveSubjectInterests = function(subjectInterests) {
```

Comment:
Something should filter this for invalid characters prior to saving.

The backend should do the same.